sdk/daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/crypto/Hash.scala

```diff
@@ -222,24 +223,68 @@ object Hash {
     }
   }
 
-  // Transaction Builder class with builder methods to recursively encode nodes
-  private sealed class TransactionBuilderV1(
```

Comment:
Got rid of the `TransactionBuilderV1` class  which wasn't adding much, `NodeBuilder` is sufficient